README.rst

```diff
@@ -45,14 +45,37 @@ The verification runs in the context of the initialized Zope application.
 Usage::
 
     ./bin/instance zodbverify [-h] [-D]
-
```

Comment:
It seems this is missing documentation of the new flag?

Reply:
I have updated the documentation by pasting the new help output.